Style attribute HTML (#1539)
Codecademy/docs: content/html/concepts/attributes/terms/style/style.md

---
Title: 'Style'
Description: 'Specifies how a page will be rendered in a web browser.'
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Style' 
  - 'Attributes'
  - 'Elements'
  - 'Tags'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

The `style` attribute specifies how an element will be rendered in a web browser, e.g. the color or the size of a font in a paragraph.

When the `style` attribute is used inline, it overrides any styles set globally. 

> >**Note:** Most developers refrain from using the inline `style` attribute and set the style of the elements through an external CSS style sheet, which helps to keep the code clear and more easily maintained. The style can also be embedded in the `<head>` element using the `<style>` tag.

Reviewer: ```suggestion
> **Note:** Most developers refrain from using the inline `style` attribute and set the style of the elements through an external CSS style sheet, which helps to keep the code clear and more easily maintained. The style can also be embedded in the `<head>` element using the `<style>` tag.
```

## Syntax

```pseudo
<element_tag style="property1:value;property2:value;"></element_tag>
```

> Like all global attributes, the `style` attribute supports all HTML tags and can take many properties, e.g. the `style` attribute can be used in the `<p>`, the `<h1>` or the `<hr>`.

Reviewer: ```suggestion

Like all global attributes, the `style` attribute supports all HTML tags and can take many properties, e.g. the `style` attribute can be used in the `<p>`, the `<h1>` or the `<hr>`.
```

Author: Done

---
Title: 'Style'
Description: 'Specifies how a page will be rendered in a web browser.'
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Style'
  - 'Attributes'
  - 'Elements'
  - 'Tags'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> The `style` attribute specifies how an element will render in a web browser, e.g., the color or size of a font in a paragraph.

Reviewer 2: ```suggestion
The **`style`** attribute specifies how an element will render in a web browser, e.g., the color or size of a font in a paragraph.
```

Author: done

> When the **`style`** attribute is used inline, it overrides any styles set globally.

Reviewer 2: ```suggestion
When the `style` attribute is used inline, it overrides any styles set globally.
```

> > **Note:** Most developers refrain from using the inline `style` attribute and set the style of the elements through an external [CSS](https://www.codecademy.com/resources/docs/css) style sheet, which helps to keep the code clear and more easily maintained. The style can also be embedded in the `<head>` element using the `<style>` tag.

Reviewer 2: ```suggestion
> **Note:** Most developers refrain from using the inline `style` attribute and set the style of the elements through an external [CSS](https://www.codecademy.com/resources/docs/css) style sheet, which helps to keep the code clean and more easily maintained. The styling of a page or elements on the page can also be embedded in the `<head>` element using the `<style>` tag.
```

Author: Done

> > **Note:** Most developers refrain from using the inline **`style`** attribute and set the style of the elements through an external [CSS](https://www.codecademy.com/resources/docs/css) style sheet, which helps to keep the code clean and more easily maintained. The styling of a page or elements on the page can also be embedded in the `<head>` element using the `<style>` tag.

Reviewer: ```suggestion
> **Note:** The styling of a page or elements on the page can also be embedded in the `<head>` element using the [`<style>`](https://www.codecademy.com/resources/docs/html/elements/style) tag. However, most developers refrain from using the inline `style` attribute and set the style of the elements through an external [CSS](https://www.codecademy.com/resources/docs/css) style sheet. This helps to keep the code clean and is more easily maintained.
```

## Syntax

```pseudo

> <element_tag style="property1:value;property2:value;"></element_tag>

Reviewer 2: ```suggestion
<element_tag style="property1:value; property2:value;"></element_tag>
```

Author: Done

```

> Like all global attributes, the **`style`** attribute supports all HTML tags and can take many properties, e.g., the **`style`**git attribute can be used in the `<p>`, [`<h1>`](https://www.codecademy.com/resources/docs/html/elements/h1-h6), or [`<hr>`](https://www.codecademy.com/resources/docs/html/elements/hr) tags.

Reviewer 2: ```suggestion
Like all global attributes, the `style` attribute supports all HTML tags and can take many properties, e.g., the `style` attribute can be used in the `<p>`, [`<h1>`](https://www.codecademy.com/resources/docs/html/elements/h1-h6), or [`<hr>`](https://www.codecademy.com/resources/docs/html/elements/hr) tags.
```

## Example

In the example below, the `<p>` element is given a `style` attribute to specify its color, the text alignment, and the font size:

```html

> <p style="color:green;font-size:12px;text-align:center;">

Reviewer 2: ```suggestion
<p style="color:green; font-size:12px; text-align:center;">
```

Author: Done
